Assets/Code/Character/ThoughtBubble.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class ThoughtBubble : MonoBehaviour
+{
+  
+  [SerializeField] private bool _Active;
+  [SerializeField] private string _startingThoughtBubbleText;
+  [SerializeField] private GameObject _thoughtBubbleRoot;
+
+  [SerializeField] private TMP_Text _thoughtBubbleTextComponent;
+  [SerializeField] private Image _thoughtBubbleSprite;
+  private string _thoughtBubbleText;
+  private bool _active;
```

Comment:
I think you have an extra "active" boolean here. You can just use the `_Active` as your c# property "backing field", as they call it.

Reply:
Fixed. This was only for debug purposes anyway.